cmd/bb_storage/main.go

```diff
@@ -25,8 +33,12 @@ import (
 
 func main() {
 	var (
-		blobstoreConfig  = flag.String("blobstore-config", "/config/blobstore.conf", "Configuration for blob storage")
-		webListenAddress = flag.String("web.listen-address", ":80", "Port on which to expose metrics")
+		blobstoreConfig      = flag.String("blobstore-config", "/config/blobstore.conf", "Configuration for blob storage")
+		webListenAddress     = flag.String("web.listen-address", ":80", "Port on which to expose metrics")
+		agentEndpointURI     = flag.String("jaeger.agent-endpoint", "127.0.0.1:6831", "jaeger agent address")
+		collectorEndpointURI = flag.String("jaeger.collector-endpoint", "http://127.0.0.1:14268/api/traces", "jaeger collector endpoint")
+		serviceName          = flag.String("service-name", "bb_storage", "service name for tracing")
```

Comment:
These flags are also specific to Jaeger, right? Could we prefix those with `jaeger.` as well?

While there, please start sentences with uppercase.